cmd/github-migration/v1/github-migration.go

```diff
@@ -38,28 +41,75 @@ func Execute() { // hello
 		// sync
 		appSync        = app.Command("sync", "Sync Bitbucket and GitHub repositories")
 		appSyncRunOnce = appSync.Flag("run-once", "Syncs the repositories once").Bool()
-		// appSyncPersonalAccount    = appSync.Flag("personal-account", "Migrates/Syncs the repositories to personal GitHub account").Bool()
+		appSyncPersonalAccount    = appSync.Flag("personal-account", "Migrates/Syncs the repositories to personal GitHub account").Bool()
 		appSyncBlockNewMigrations = appSync.Flag("block-new-migrations", "Block new migrations and sync only existing repos on GitHub").Bool()
 
 		/////////
 		// interactive - Can leave it out, does not make sense if supporting multiple sources for integrations
 		appInteractive = app.Command("interactive", "Select the projects and repositories to migrate/sync")
+
+		/////////
+		// test
+		appTest = app.Command("test", "Test out new features")
 	)
 
 	p := kingpin.MustParse(app.Parse(os.Args[1:]))
 
+	config := config.Parse()
+	fmt.Printf("%+v\n", config)
+
 	switch p {
 
 	case appSync.FullCommand():
-		fmt.Println("SYNC")
-		fmt.Printf("App Log Level: %v\n", *appLogLevel)
-		fmt.Printf("Run Once: %v\n", *appSyncRunOnce)
-		// fmt.Printf("Personal Account: %v\n", *appSyncPersonalAccount)
-		fmt.Printf("Block New: %v\n", *appSyncBlockNewMigrations)
+		log.Infof("SYNC")
+		log.Infof("App Log Level: %v\n", *appLogLevel)
```

Comment:
Logrus itself has built-in support for structured logging.  Would be nice to use it in a future PR.

https://github.com/sirupsen/logrus

Reply:
👍 I'll switch to structured logging in a future PR